pandas/plotting/_core.py

```diff
@@ -1935,52 +1935,134 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
 
 
 _shared_docs['boxplot'] = """
-    Make a box plot from DataFrame column optionally grouped by some columns or
-    other inputs
+    Make a box-and-whisker plot from DataFrame column optionally grouped
+    by some columns or other inputs. The box extends from the Q1 to Q3
+    quartile values of the data, with a line at the median (Q2).
+    The whiskers extend from the edges of box to show the range of the data.
+    Flier points (outliers) are those past the end of the whiskers.
+    The position of the whiskers is set by default to 1.5 IQR (`whis=1.5``)
+    from the edge of the box.
+
+    For further details see
+    Wikipedia's entry for `boxplot <https://en.wikipedia.org/wiki/Box_plot/>`_.
 
     Parameters
     ----------
-    data : the pandas object holding the data
     column : column name or list of names, or vector
-        Can be any valid input to groupby
+        Can be any valid input to groupby.
     by : string or sequence
-        Column in the DataFrame to group by
-    ax : Matplotlib axes object, optional
+        Column in the DataFrame to groupby.
+    ax :  Matplotlib axes object, (default `None`)
+        The matplotlib axes to be used by boxplot.
     fontsize : int or string
+        The font-size used by matplotlib.
     rot : label rotation angle
+        The rotation angle of labels.
+    grid : boolean( default `True`)
+        Setting this to True will show the grid.
     figsize : A tuple (width, height) in inches
-    grid : Setting this to True will show the grid
+        The size of the figure to create in inches by default.
     layout : tuple (optional)
-        (rows, columns) for the layout of the plot
+        Tuple (rows, columns) used for the layout of the plot.
     return_type : {None, 'axes', 'dict', 'both'}, default None
         The kind of object to return. The default is ``axes``
         'axes' returns the matplotlib axes the boxplot is drawn on;
```

Comment:
They way this is written makes it render in the documentation in a confusing way: "The default is axes `axes`".

Also, probably is better to find a way to explain the different options in different lines (like bullet points) instead of separating the different options by a semicolon only.